nx.json

```diff
@@ -26,12 +27,9 @@
       }
     }
   },
-  "targetDependencies": {
-    "build": [
-      {
-        "target": "build",
-        "projects": "dependencies"
-      }
-    ]
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"]
```

Comment:
This is just a more concise way of expressing the same thing.

The `^` means "of my dependencies".

So here `build` tasks depend on the `build` tasks of their dependencies